dashboard/client/src/components/FunctionOverviewPanel/FunctionOverviewPanel.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Card, CardHeader, CardBody, Row, Col } from 'reactstrap';
+
+const FunctionOverviewPanel = ({ children, headerText, headerIcon, button, className, bodyClassName }) => {
+  return (
+    <Card className={`height-100 ${className}`}>
+      <CardHeader
+        tag="h6"
+        className="flex align-items-center justify-content-space-between min-height-49"
+      >
+        <span>
+          { headerIcon }
+          { headerText }
+        </span>
+        { button }
+      </CardHeader>
+      <CardBody className={bodyClassName}>
+        { children }
+      </CardBody>
+    </Card>
+  );
+};
+
+const getOppositeSize = (size) => {
+  if (size >= 12 || size <= 0) {
+    return size;
+  }
+
+  return 12 - size;
+};
+
+const MetaList = ({ list, sizes }) => {
+  let currentSizes = sizes;
+
+  if (sizes === undefined) {
+    currentSizes = {
+      xs: 12,
+      sm: 3,
+      md: 2,
+      lg: 3,
+      xl: 2,
+    }
+  }
+
+  return (
+    <div>
+      { list.map(({ label, value, renderValue }) => {
+        const { xs, sm, md, lg, xl } = currentSizes;
+
+        return (
+          <Row noGutters key={label} className="py-1">
+            <Col xs={xs} sm={sm} md={md} lg={lg} xl={xl} className="is-bold">
+              { label }
+            </Col>
+            <Col
+              xs={getOppositeSize(xs)}
```

Comment:
`getOppositeSize` ?

Reply:
There are 12 columns in layout. If `xs` == `4` this function need to return `8` (12 - 4) . Just helper function

Reply:
Can you document the function please? `getOppositeSize` doesn't self-describe.